content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -75,6 +75,34 @@ appending a container name to the command, with a `-c` flag, like so:
 kubectl logs counter -c count
 ```
 
+Since v1.32, you can request a specific log stream of a container in a running Pod.
+This functionality requires the
+`PodLogsQuerySplitStreams` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
+Unfortunately, only the server side changes have been merged so far; the [corresponding kubectl changes](https://github.com/kubernetes/kubernetes/pull/128159) are still under review.
+In the meantime, you can still use this feature by directly accessing the API.
+
+To demonstrate this feature, you can create a Pod that periodically writes text to both the standard output and error stream.
+
+{{% code_sample file="debug/counter-pod-err.yaml" %}}
+
+To run this pod, use the following command:
+
+```shell
+kubectl apply -f https://k8s.io/examples/debug/counter-pod-err.yaml
+```
+
+To fetch only the stderr log stream, you can run:
+
+```shell
+kubectl get --raw "/api/v1/namespaces/default/pods/counter-err/log?stream=Stderr"
+```
+
+Once kubectl is updated, you will be able to easily retrieve a specific log stream using:
+
+```shell
+kubectl logs --stream=stderr counter-err
+```
```

Comment:
We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) in docs, so:
```suggestion
```